io/CopcReader.hpp

```diff
@@ -53,6 +53,7 @@ namespace las
     struct Header;
     struct Vlr;
     class VlrCatalog;
+    using VlrList = std::vector<Vlr>;
```

Comment:
Shouldn't this be in somewhere more central? I know it's just a declaration here though.